docs/environment.yaml

```diff
@@ -20,3 +20,4 @@ dependencies:
   - sphinx-design
   - sphinx-toolbox
   - git+https://github.com/OpenFreeEnergy/gufe@main
+  - git+https://github.com/OpenFreeEnergy/Lomap@main
```

Comment:
Is it time for another lomap release?

Reply:
There are concerns about backward compatibility breaks for current users. We need to give them (namely, Mary) a migration window.

Reply:
if we can't do a release right now then we can't merge this if want to have a release by end of week.

Reply:
Fine by me. We need to get a migration writeup in place, IMO. I'll remove this from the 0.12.0 milestone.

Reply:
Fine with moving this to 0.12 but I don't understand:

> There are concerns about backward compatibility breaks for current users. We need to give them (namely, Mary) a migration window.

'lomap2 < $NEW_VERSION'

Isn't enough